README.md

```diff
@@ -60,3 +66,27 @@ Steps:
 1. Click the ``Open in Gitpod`` button.
 2. Relax: a development environment with an active Wagtail Guide site will be created for you.
 3. Login at `/admin/` with username `admin` and password `changeme`
+
+
+# Contributing
+
+If you're a Python or Django developer, fork the repo and join us. You'll find answers to many common new contributor questions in our [contributing guidelines](https://github.com/wagtail/bakerydemo/blob/master/contributing.md).
```

Comment:
Why do we link to the contributing guide of bakerydemo?

We should add `CONTRIBUTING.md` to this project.

Reply:
Because apparently I didn't update the link to the main Wagtail contributing guidelines when I copied the README structure over that I liked from the bakerydemo. We can definitely add a CONTRIBUTING.md.